### DISCUSSION TITLE
I can't access my website anymore

### DISCUSSION THREAD

**Discussion by itwxb** [vercel/community#3880] [Bug Report]
### Question

Hello, I can't access my website anymore. I am located in Henan Province, China. I have already resolved the domain name to a Chinese IP address. However, I was able to access it yesterday, but not today. My friend in another province can access it. What could be the reason? The website address is qmixi.cn.

**Comment by amyegan** [comment]
Hey @itwxb. That sounds like a network problem, which may be temporary. Do you still have this problem?

**Reply by itwxb** [reply]
It's been two days and I still can't access, I'm in Henan Province, China. I don't think it's because of me, because my friend can't access it in Henan, can you help me, I like the platform of vercel very much! Thanks!
